README.md

```diff
@@ -3,28 +3,29 @@
 ## Quick Install (Project Setup)
 
 1.  `$ git clone git@github.com:BridgeCityBicycleCoop/workstand.git`
-2. `$ cd workstand/bikeshop_project`
-3. `$ npm install`
-4. `$ docker-machine start && eval $(docker-machine env)`
-5. `$ cd .. && docker-compose -f docker-compose.yml -f docker-compose.dev.yml build`
-6. `$ docker-compose -f docker-compose.yml -f docker-compose.dev.yml run workstand python manage.py migrate`
-7. `$ docker-compose -f docker-compose.yml -f docker-compose.dev.yml run workstand python manage.py loaddata fixtures.yaml`
-8. `docker-compose -f docker-compose.yml -f docker-compose.dev.yml up`
-9. Visit http://192.168.99.100:8000/ (`docker-machine ls` to see IP address)
-10. Login with `u: admin@workstand.dev`and `p: bike!bike!`
-11. Stop docker any time with `docker-compose -f docker-compose.yml -f docker-compose.dev.yml down`
+2. `$ cd workstand`
+3. `$ brew install pyenv`
+4. `$ brew install pipenv`
+5. `$ pyenv install 3.6.6`
```

Comment:
Getting an install error here: `zipimport.ZipImportError: can't decompress data; zlib not available`

Reply:
Seems to be fixed with this command:
`sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /`

Found via: https://stackoverflow.com/questions/52741673/how-can-i-install-zlib-on-mac-os-x-mojave-10-14

May want to make note about it?